core/src/main/python/wlsdeploy/util/cla_utils.py

```diff
@@ -102,7 +102,7 @@ class CommandLineArgUtil(object):
     USAGE_ERROR_EXIT_CODE          = 99
     ARG_VALIDATION_ERROR_EXIT_CODE = 98
     PROG_RESTART_REQUIRED          = 103
-    PROG_ROLLBACK_IF_RESTART_EXIT_CODE = 3
+    PROG_ROLLBACK_IF_RESTART_EXIT_CODE = 104
```

Comment:
Needs a message in shared.sh, shared.cmd in checkExitCode for this exit code

Reply:
fixed